packages/consensus/source/consensus.ts

```diff
@@ -320,6 +327,11 @@ export class Consensus implements Contracts.Consensus.IConsensusService {
 		}
 		this.logger.info(`Received +2/3 precommits for ${this.#height}/${this.#round} blockId: ${block.data.id}`);
 
+		if (roundState instanceof RoundState) {
```

Comment:
@sebastijankuzner Not sure if this is the best way to do it, but this check is needed since we also end up here during p2p download in which case we don't have an actual round state instance.